app/src/main/res/drawable/button_blue.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<shape
+    xmlns:android="http://schemas.android.com/apk/res/android">
+    <stroke
+        android:width="1dp"
```

Comment:
Solve harcoded dimens and colors! There are files like dimens.xml and colors.xml where you can save them (there are some already saved)